client/components/Button/Button.css

```diff
@@ -13,4 +13,8 @@
     margin-left: calc($base-spacing * 2);
     box-shadow: $button-shadow;
   }
+  &[disabled] {
+    background: $grey;
```

Comment:
are you sure you want to use `background` here?

Reply:
Change to background-color